### PR TITLE
build: run erlfmt with files name by `$(FIND)`

### PR DESCRIPTION
**Pull request by JimMoen**
~Fixes~
**NO** Production code changed.

~Release version: v/e5.?~

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_dashboard/src/emqx_dashboard.appup.src [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  %% we should always restart dashboard to make sure api rules/swagger is updated
-  [{<<".*">>,[{restart_application, emqx_dashboard}]}],
```
Comment (zmstone) [inline apps/emqx_dashboard/src/emqx_dashboard.appup.src:5]: ?

**Comment by zmstone** [comment]
how much faster ?

**Comment by JimMoen** [comment]
on master:
```bash
$ time make fmt
make fmt  25.88s user 6.23s system 845% cpu 3.798 total
```

After changed.
```bash 
$ time make fmt
make fmt  22.69s user 5.32s system 756% cpu 3.701 total
```

File: apps/emqx_dashboard/src/emqx_dashboard.appup.src (hunk 1 above)
Comment (JimMoen) [reply apps/emqx_dashboard/src/emqx_dashboard.appup.src:5]: erlfmt changed it.
should we remove all `*.appup.src` ?

Comment (zmstone) [reply apps/emqx_dashboard/src/emqx_dashboard.appup.src:5]: yes, let's exclude appup.src files.

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-3364)

**Pull request merged**